DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
 
 If you want to read more into `flatpak override`, you can look at the [`flatpak-override` documentation](https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override).
 
-
 ### Permissions
 
+This is a list of permissions that can be configured using Flatseal. The descriptions below assume that the options are enabled (toggled on) when it's a toggle.
+
+There is also a "`flatpak override` equivalent" row for those that want to get started on the CLI of Flatpak permissions.
+
 #### Share
 
 List of subsystems shared with the host system.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-Network | Toggle | Allow or prohibit access to the network. | `--share=network` and `--unshare=network`
-[Inter-process communications](https://en.wikipedia.org/wiki/Inter-process_communication) | Toggle | Share or unshare IPC namespace with the host. | `--share=ipc` and `--unshare=ipc`
+Network | Toggle | Allow the application to have access to the network. <br /> <br /> For example, if it's disabled for Firefox, you will no longer be able to browse the internet. | `--share=network` and `--unshare=network`
+[Inter-process communications](https://en.wikipedia.org/wiki/Inter-process_communication) | Toggle | Share IPC namespace with the host. <br /> <br /> This is required by X11 due to it depending on IPC. Even if it's enabled, this will not make it any more dangerous. | `--share=ipc` and `--unshare=ipc`
```

Comment:
Please remove

>  Even if it's enabled, this will not make it any more dangerous. 

I don't think is adding too much extra information, and better to keep it short.